C23_ObjectRecognition/TBD min_dimensions
C23_ObjectRecognition/TBD max_dimensions
